internal/docker/metadata.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2022 The Witness Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docker
```

Comment:
Does this need to be an internal package? Are these redeclarations of the structs in the docker attestor?

Reply:
so, the main reason I put it in internal is that it's go structs that we have defined to unmarshal the metadata file that docker. Generates. The most obvious thing to do would be to use docker's structs, but they don't exist, from what I can tell the file is created by using unstructured `map[string]interface{}`. We don't want people to depend on these structs, so I put them in internal.